internal/installer/elasticagent_zip.go

```diff
@@ -122,13 +123,22 @@ func (i *elasticAgentZIPPackage) Preinstall(ctx context.Context) error {
 		return err
 	}
 
-	_, err = i.Exec(ctx, []string{"powershell.exe", "Expand-Archive", "-LiteralPath", binaryPath, "-DestinationPath", "C:\\", "-Force"})
-	if err != nil {
-		return err
-	}
+	output, err := i.Exec(ctx, []string{"powershell.exe", "Test-Path", "C:\\elastic-agent"})
+	log.WithFields(log.Fields{
+		"output": output,
+		"error":  err}).Trace("Checking for existence of elastic-agent installation directory")
```

Comment:
NIT: consistent indentation
```suggestion
	log.WithFields(log.Fields{
		"output": output,
		"error":  err
	}).Trace("Checking for existence of elastic-agent installation directory")
```